frontend/components/Footer.react.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+class Footer extends React.Component{
```

Comment:
Denne burde skrive om til en [stateless component] (https://egghead.io/lessons/react-building-stateless-function-components-new-in-react-0-14)